### PR TITLE
feat: add solana tokens to coins

### PR DESCRIPTION
**Pull request by 0xksure**
## Add Solana token support to coins

### CODE REVIEW

File: src/coins.ts [hunk 1]
```diff
@@ -1201,6 +1316,16 @@ export const wrappedTokens: { [ChainId: string]: Token } = {
...
   },
+  [ChainId.SOL]: {
+    address: 'So11111111111111111111111111111111111111112',
+    symbol: 'SOL',
+    decimals: 9,
+    chainId: ChainId.SOL,
+    coinKey: CoinKey.SOL,
+    name: 'Wrapped SOL',
+    logoURI:
+      'https://assets.coingecko.com/coins/images/4128/small/solana.png?1640133422',
```
Comment (chybisov) [inline src/coins.ts:1327]: Do we really need these params `?1640133422` in the end? 🙂

**Review by chybisov — CHANGES_REQUESTED** [review]

File: src/coins.ts (hunk 1 above)
Comment (0xksure) [reply src/coins.ts:1327]: haha no

Comment (chybisov) [reply src/coins.ts]: @0xksure let's remove them from all the tokens 😄

File: src/coins.ts [hunk 2]
```diff
@@ -1162,6 +1221,60 @@ const basicCoins: BasicCoin[] = [
       },
     },
   },
+  // > Solana
+  {
+    key: CoinKey.SOL,
+    name: CoinKey.SOL,
+    logoURI:
+      'https://assets.coingecko.com/coins/images/4128/small/solana.png?1640133422',
```
Comment (chybisov) [inline src/coins.ts:1229]: Same `?1640133422`

**Review by chybisov — CHANGES_REQUESTED** [review]

**Review by chybisov — APPROVED** [review]

**Pull request merged**